src/java/com/unifina/utils/testutils/FakeStreamService.java

```diff
@@ -21,4 +22,13 @@ public void sendMessage(Stream stream, Map message) {
 		sentMessagesByChannel.get(c).add(message);
 	}
```

Comment:
`sendMessage(Stream stream, Map message)` doesn't exist anymore in `StreamService`, safe to remove? (It should actually fail compilation due to `@Override` annotation, but it's a Groovy bug that it doesn't fail)